core/src/main/java/io/grpc/ManagedChannelBuilder.java

```diff
@@ -437,7 +437,7 @@ public T disableRetry() {
   }
 
   /**
-   * Enables the retry mechanism provided by the gRPC library.
+   * Enables the retry or hedging mechanism provided by the gRPC library.
```

Comment:
I think this should be "and hedging mechanisms" instead of "or"

Reply:
Done.